english/froboznia/froboznia.md

```diff
@@ -1,3 +1,10 @@
 You find yourself face to face with the Froboznian police.
 
 "Ouzna gotcha!" they bellow.
+
+You decide to
+
+-> [Surrender](Surrender\Surrender.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.